src/explorer.api/Controllers/ExploreController.cs

```diff
@@ -80,7 +76,20 @@ public IActionResult Result(System.Guid exploreId)
         {
             if (Explorers.TryGetValue(exploreId, out var explorer))
             {
-                return Ok(explorer.LatestResult);
+                var exploreStatus = explorer.Completion().Status switch
+                {
+                    TaskStatus.RanToCompletion => ExploreResult.ExploreStatus.Complete,
+                    TaskStatus.Running => ExploreResult.ExploreStatus.Processing,
+                    _ => ExploreResult.ExploreStatus.Error,
+                };
+
+                var metrics = explorer.ExploreMetrics
+                    .Select(m => new ExploreResult.Metric(m.Name, m.Metric));
+
+                return Ok(new ExploreResult(
```

Comment:
I thought about this but then I figured that it might be a nice feature to be able to re-query for the same result set. But I think you're right, better to reclaim the memory. If the above 'feature' is ever needed, we can add it.

Reply:
5c41e3e442e2f1382d9f29586dda98759820b189

Reply:
Keeping up a cache of old results sounds like an interesting idea, I didn't thought about it (it is similar to the VCR). Maybe we should add a new issue (enhancement/feature) for it.